reactor-netty-core/src/test/java/reactor/netty/resources/ConnectionProviderTest.java

```diff
@@ -32,9 +33,21 @@
 
 class ConnectionProviderTest {
 
+	static final ConnectionProvider.MeterRegistrar METER_REGISTRY = new ConnectionProvider.MeterRegistrar() {
```

Comment:
Once we have the new method with a default implementation we can revert the changes in this class.